test/py/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ add_test(NAME python_test_generate
 )
 
 add_test(NAME python_test
-    COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/RunClientServer.py --gen-base=${CMAKE_CURRENT_BINARY_DIR}
+    COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/RunClientServer.py --gen-base=${CMAKE_CURRENT_BINARY_DIR} --msvc-profile=${MSVC_PROFILE}
```

Comment:
@jeking3 By the way, I also disabled my Python tests. Here are the places I modified to disable a test suite in Python.